src/modules/NetworkStorage.rb

```diff
@@ -39,21 +42,23 @@ def main
     # Ask /proc/mounts what device a mount point is using.
     # @return e.g. /dev/sda2 (or just "nfs")
     def getDevice(mount_point)
-      cmd = Builtins.sformat(
```

Comment:
in general if you write tests and do refactoring it is good to split it into two commits, so it is easily verifiable that tests passed even before refactoring